### DISCUSSION TITLE
How to track video plays with plausible on a WordPress site?

### DISCUSSION THREAD

**Discussion by Schokolader** [plausible/analytics#2443] [Anything Else]
Hello,

How to track video plays with plausible on a WordPress site? 

Thank you

**Comment by metmarkosaric** [comment]
hi @Schokolader! you would need to set up custom events for that. we have some instructions for tracking of audio and video plays specifically here: https://plausible.io/docs/custom-event-goals#tracking-audio-and-video-elements

**Reply by Schokolader** [reply]
Thanks a lot. Didn't saw this.
